src/app/DTT/select/index.js

```diff
@@ -167,6 +171,7 @@ const Write = ({
         useBasicStyles
         isMulti={isMulti}
         options={options}
+        formatOptionLabel={formatOptionLabel}
```

Comment:
Why are we passing formatOptionLabel? What does passing formatOptionLabel as a function to Select component do?

Reply:
Is it so that we can access the test id in the options?

Reply:
Yes, formatOptionLabel overrides the default select item component, allowing us to pass through a component with a test-id